e2e/pages/groupForm.po.js

```diff
@@ -14,5 +15,11 @@ class GroupForm {
                 groupSetter[field].call();
             });
     }
+
+    clickCreateButton() {
+        CommonActions.click('input#edit-submit');
+        browser.waitForVisible('#popups-overlay', 30000, true);
```

Comment:
@mgarciabustamante  better to create a method within CommonActions with descriptive name in order to improve readability. example:
 CommonActions.waitForNotVisible(locator);